### PR TITLE
Niamh25 doc 1854 update summary for default class evolution

### PR DESCRIPTION
**Pull request by Niamh25**
Added to summary and added a note regarding contractor properties. Amos recommended that it be added to this section to say explicitly what is not supported after the section mentioning what is supported.

### CODE REVIEW

File: content/en/platform/corda/4.10/enterprise/serialization-default-evolution.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ Corda’s serialization framework supports minor modifications to default classe
 * Remove properties.
 * Reorder constructor parameters.
 
+The following are not supported:
+
+* Renaming existing constructor properties.
+* Changing the type of existing constructor parameters.
+
+{{< note >}}
+Any transient properties that aren't declared in the constructor are not serialized.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.10/enterprise/serialization-default-evolution.md:33]:
```suggestion
Any transient properties that are not declared in the constructor are not serialized.
```

File: content/en/platform/corda/4.7/enterprise/serialization-default-evolution.md [hunk 2]
```diff
@@ -21,14 +21,19 @@ Whilst more complex evolutionary modifications to classes require annotating, Co
...
 * Removing properties
 * Reordering constructor parameters
 
+The following are not supported:
+
+* Renaming existing constructor properties.
+* Changing the type of existing constructor parameters.
 
+{{< note >}}
+Any transient properties that aren't declared in the constructor are not serialized.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.7/enterprise/serialization-default-evolution.md:35]:
```suggestion
Any transient properties that are not declared in the constructor are not serialized.
```

File: content/en/platform/corda/4.8/enterprise/serialization-default-evolution.md [hunk 3]
```diff
@@ -24,6 +24,14 @@ Corda’s serialization framework supports minor modifications to default classe
 * Remove properties.
 * Reorder constructor parameters.
 
+The following are not supported:
+
+* Renaming existing constructor properties.
+* Changing the type of existing constructor parameters.
+
+{{< note >}}
+Any transient properties that aren't declared in the constructor are not serialized.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.8/enterprise/serialization-default-evolution.md:33]:
```suggestion
Any transient properties that are not declared in the constructor are not serialized.
```

File: content/en/platform/corda/4.9/enterprise/serialization-default-evolution.md [hunk 4]
```diff
@@ -24,6 +24,14 @@ Corda’s serialization framework supports minor modifications to default classe
 * Remove properties.
 * Reorder constructor parameters.
 
+The following are not supported:
+
+* Renaming existing constructor properties.
+* Changing the type of existing constructor parameters.
+
+{{< note >}}
+Any transient properties that aren't declared in the constructor are not serialized.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.9/enterprise/serialization-default-evolution.md:33]:
```suggestion
Any transient properties that are not declared in the constructor are not serialized.
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thnx @Niamh25

**Pull request merged**